docs/guides/component-testing/getting-started.mdx

```diff
@@ -388,7 +393,7 @@ more info on writing good selectors, see our guide
 
 ### Passing Props to Components
 
-We should also have a test to ensure the `initial` prop sets the count to
+We should also have a test to ensure the `count` prop sets the count to
```

Comment:
This doesn't work with https://github.com/cypress-io/component-testing-quickstart-apps/blob/main/react/my-awesome-app/src/components/Stepper.jsx as it is currently defined.